src/main/java/picard/fingerprint/CrosscheckFingerprints.java

```diff
@@ -253,6 +263,13 @@ public class CrosscheckFingerprints extends CommandLineProgram {
             doc = "One or more input files (or lists of files) with which to compare fingerprints.", minElements = 1)
     public List<String> INPUT;
 
+    @Argument(doc = "A tsv with two columns and no header which maps the input files to corresponding indices; to be used when index " +
+            "files are not located next to input files. First column must match the list of inputs. ", optional = true)
+    public File INPUT_INDEX_MAP;
+
+    @Argument(doc = "A boolean value to determine whether input files should only be parsed if index files are available.")
```

Comment:
Maybe add a sentence here mentioning that it is potentially much slower to run this tool without index files? Basically giving some indication of why a user might choose to use this option.

Reply:
Good point. Added!